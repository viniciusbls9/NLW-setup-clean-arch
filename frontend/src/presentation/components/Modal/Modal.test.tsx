import { describe, test, expect } from 'vitest';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Modal } from '.';

describe('Header Component', () => {
  test('Should render correctly modal component', () => {
    const { container } = render(<Modal />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
