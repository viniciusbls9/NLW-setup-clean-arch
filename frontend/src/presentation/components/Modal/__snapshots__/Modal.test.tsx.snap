// Vitest Snapshot v1

exports[`Header Component > Should render correctly modal component 1`] = `
<button
  aria-controls="radix-:r0:"
  aria-expanded="false"
  aria-haspopup="dialog"
  class="border border-violet-500 font-semibold rounded-lg px-6 py-4 flex items-center gap-3 hover:border-violet-300"
  data-state="closed"
  type="button"
>
  <svg
    class="text-violet-500"
    fill="currentColor"
    height="20"
    viewBox="0 0 256 256"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="none"
      height="256"
      width="256"
    />
    <line
      fill="none"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="16"
      x1="40"
      x2="216"
      y1="128"
      y2="128"
    />
    <line
      fill="none"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="16"
      x1="128"
      x2="128"
      y1="40"
      y2="216"
    />
  </svg>
  Novo h√°bito
</button>
`;
